Adjust for newer PulseAudio API.
We can not use PA_API_VERSION here, since it did not change (== 12).

Lubomir Rintel <lkundrak@v3.sk>

diff -urp VirtualBox-3.0.2_OSE/src/VBox/Devices/Audio/pulse_stubs.c VirtualBox-3.0.2_OSE.fixed/src/VBox/Devices/Audio/pulse_stubs.c
--- VirtualBox-3.0.2_OSE/src/VBox/Devices/Audio/pulse_stubs.c	2009-07-10 11:41:12.000000000 -0400
+++ VirtualBox-3.0.2_OSE.fixed/src/VBox/Devices/Audio/pulse_stubs.c	2009-08-04 11:35:03.036925820 -0400
@@ -41,10 +41,17 @@ void (*function ## _fn)(void); \
 void function signature \
 { ( (void (*) signature) function ## _fn ) shortsig; }
 
+#if PA_PROTOCOL_VERSION >= 16
+PROXY_STUB     (pa_stream_connect_playback, int,
+                (pa_stream *s, const char *dev, const pa_buffer_attr *attr,
+                 pa_stream_flags_t flags, const pa_cvolume *volume, pa_stream *sync_stream),
+                (s, dev, attr, flags, volume, sync_stream))
+#else
 PROXY_STUB     (pa_stream_connect_playback, int,
                 (pa_stream *s, const char *dev, const pa_buffer_attr *attr,
                  pa_stream_flags_t flags, pa_cvolume *volume, pa_stream *sync_stream),
                 (s, dev, attr, flags, volume, sync_stream))
+#endif
 PROXY_STUB     (pa_stream_connect_record, int,
                 (pa_stream *s, const char *dev, const pa_buffer_attr *attr,
                 pa_stream_flags_t flags),
